hedyorg/hedy
# 💻 Roles of variables: add stepper in for loop

I've added the stepper role, it works with for loops, but other kinds of steppers are not working yet.

**How to test**

Follow these steps to verify this PR works as intended:

* change the feature flag showRoles in debugging.ts to true
* go to Hedy level 11 and run this program (or something similar) in the debugger:
```
for counter in range 1 to 10
    print counter
print 'Ready or not. Here I come!'
```
* check that in the variables view, the role that is shown for the variable counter is stepper. 

**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done)

- [ x] Contains one of the PR categories in the name
- [ x] Describes changes in the format above
- [ x] Has a "How to test" section

If you're unsure about any of these, don't hesitate to ask. We're here to help!
